spec/models/help_request_spec.rb

```diff
@@ -4,13 +4,13 @@
   describe 'DB table' do
     it { is_expected.to have_db_column :title }
     it { is_expected.to have_db_column :message }
-    it { is_expected.to have_db_column :status }
+    it { is_expected.to have_db_column :urgent }
   end
 
   describe 'Validations' do
     it { is_expected.to validate_presence_of :title }
     it { is_expected.to validate_presence_of :message }
-    it { is_expected.to validate_presence_of :status }
+    it { is_expected.to validate_presence_of :urgent }
```

Comment:
Test it using your client or by creating a request spec where :urgent is not being passed in (the way the client behaves at the moment) It's your call.